docs/src/pages/components/stack/stack.md

```diff
@@ -0,0 +1,13 @@
+---
+title: React Stack component
+components: Stack
+githubLabel: 'component: Stack'
+---
+
+# Stack
+
+<p class="description">The Stack component manages children along the vertical and horizontal axes.</p>
+
```

Comment:
I think that we should add `{{"component": "modules/components/ComponentLinkHeader.js"}}` at least for the "feedback" action